### PR TITLE
Quicksight

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

Add AWS Quicksight

**Note:**
- requires a [new RDS user to first be created](https://docs.google.com/document/d/1BkAkz45CVQRGQSNwN018beTebrbTUNpyTpGEaO7Clyc/edit#heading=h.mnku3slnkhf2) (done on staging)
- only plans / applies the Quicksight module in staging

### CODE REVIEW

File: aws/quicksight/data_sources.tf [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
+resource "aws_quicksight_data_source" "default" {
```
Comment (ben851) [inline aws/quicksight/data_sources.tf:2]: Can we change the resource name from "default" to something more meaningful? Even just "Notify"

File: aws/quicksight/data_sources.tf [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      password = var.quicksight_db_user_password
+    }
+  }
+  vpc_connection_properties {
+    vpc_connection_arn = aws_quicksight_vpc_connection.rds.arn
+  }
+  parameters {
+    rds {
+      database    = var.database_name
+      instance_id = "test"
```
Comment (ben851) [inline aws/quicksight/data_sources.tf:19]: What is an instance_id in this case? If it's an arbitrary value, can we sync it to var.env?

File: aws/quicksight/vpc.tf [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          ]
+          Resource = ["*"]
+        }
+      ]
+    })
+  }
+}
+
+resource "aws_quicksight_vpc_connection" "rds" {
+  vpc_connection_id  = "example-connection-id"
```
Comment (ben851) [inline aws/quicksight/vpc.tf:36]: this will need to be the vpc_id from the "common" outputs.

File: aws/quicksight/vpc.tf [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  name               = "Example Connection"
```
Comment (ben851) [inline aws/quicksight/vpc.tf:37]: Can we change this to something like "Notify VPC"

**Review by ben851 — CHANGES_REQUESTED** [review]
Very good first crack, just some small refinements to make

File: aws/quicksight/data_sources.tf (hunk 2 above)
Comment (sastels) [reply aws/quicksight/data_sources.tf:19]: hrm, when I was playing around before it was the `aws_db_instance` identifier. We have a cluster identifier now maybe? I'll try that...

**Review by ben851 — APPROVED** [review]
LGTM!

**Pull request merged**